### PR TITLE
Editorial: shortcut strings are localizable

### LINKED ISSUE #872: Should dir and lang apply to name/description of app shortcuts?
The [dir](https://www.w3.org/TR/appmanifest/#dir-member) and [lang](https://www.w3.org/TR/appmanifest/#lang-member) members specify the base direction and language for the [directionality-capable members](https://www.w3.org/TR/appmanifest/#dfn-directionality-capable-members) of the manifest. The [directionality-capable members](https://www.w3.org/TR/appmanifest/#dfn-directionality-capable-members) are:

* [description](https://www.w3.org/TR/appmanifest/#description-member) member
* [name](https://www.w3.org/TR/appmanifest/#name-member) member
* [short_name](https://www.w3.org/TR/appmanifest/#short_name-member) member

I wonder if they apply to `name`, `short_name`, and `description` of [ShortcutItem](https://www.w3.org/TR/appmanifest/#shortcutitem-and-its-members) as well, since they're different from the members of the manifest itself (`WebAppManifest` dictionary).

### PR DESCRIPTION
**Pull request by aarongustafson**
Closes #872

This change (choose one):

* [ ] Breaks existing normative behavior (please add label "breaking")
* [ ] Adds new normative requirements
* [ ] Adds new normative recommendations or optional items
* [ ] Makes only editorial changes (only changes informative sections, or
  changes normative sections without changing behavior)
* [ ] Is a "chore" (metadata, formatting, fixing warnings, etc).

Implementation commitment (delete if not making normative changes):

* [ ] Safari (link to issue)
* [ ] Chrome (link to issue)
* [ ] Firefox (link to issue)
* [ ] Edge (public signal)

Commit message:

Per #872, implementors are parsing `lang` and `dir`, but no one (that we know of) is implementing directionality, so I have left "Implementation commitment" blank for now.

***
<a href="https://pr-preview.s3.amazonaws.com/aarongustafson/manifest/pull/897.html" title="Last updated on Jun 10, 2020, 4:46 AM UTC (b0c38c7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/897/b7bb380...aarongustafson:b0c38c7.html" title="Last updated on Jun 10, 2020, 4:46 AM UTC (b0c38c7)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1666,11 +1666,20 @@ <h3>
           <li>
             <a>short_name</a> member.
           </li>
+          <li>
+            <a>ShortcutItem.name</a> member.
```
Comment (marcoscaceres) [inline index.html:1670]:
```suggestion
            {{ShortcutItem}}'s {{ShortcutItem/name}} member.
```

File: index.html [hunk 2]
```diff
@@ -1666,11 +1666,20 @@ <h3>
...
+          </li>
+          <li>
+            <a>ShortcutItem.short_name</a> member.
```
Comment (marcoscaceres) [inline index.html:1673]:
```suggestion
            {{ShortcutItem}}'s {{ShortcutItem/short_name}} member.
```

File: index.html [hunk 3]
```diff
@@ -1666,11 +1666,20 @@ <h3>
...
+          </li>
+          <li>
+            <a>ShortcutItem.description</a> member.
```
Comment (marcoscaceres) [inline index.html:1676]:
```suggestion
            {{{{ShortcutItem}}'s {{ShortcutItem/description}} member.
```

File: index.html [hunk 4]
```diff
@@ -1649,12 +1649,12 @@ <h3>
...
         <p>
           The <dfn>dir</dfn> member specifies the <dfn>base direction</dfn> for
-          the <a>directionality-capable members</a> of the <a>manifest</a>. The
+          the <a>localizable string members</a> of the <a>manifest</a>. The
           <a>dir</a> member's value can be set to one of the <a>text-direction
           values</a>.
         </p>
         <p>
-          The <dfn>directionality-capable members</dfn> are:
+          The <dfn>localizable string members</dfn> are:
```
Comment (marcoscaceres) [inline index.html:1657]: Maybe just? 
```suggestion
          The <dfn>localizable members</dfn> are:
```

**Comment by aarongustafson** [comment]
@marcoscaceres Accepted & merged

**Comment by aarongustafson** [comment]
@marcoscaceres Those validation error messages are not me, right? What do we need to fix?

**Comment by marcoscaceres** [comment]
> @marcoscaceres Those validation error messages are not me, right? What do we need to fix?

Nope, they are https://github.com/w3c/manifest/issues/884 ... working on it behind the scenes.

**Pull request merged**